en/08_Changelogs/6.0.0.md

```diff
@@ -111,9 +110,7 @@ The code in `silverstripe/externallinks`, `silverstripe/security-report`, and `s
 
 CMS 6 compatible versions of `silverstripe/blog`, `silverstripe/subsites`, and `silverstripe/crontask` have been released with CMS 6.0.0, though they are not commercially supported.
 
-### Campaign admin removed from the CMS {#campaign-admin-removed}
-
-The `silverstripe/campaign-admin` was a core module that provided a way to publish multiple related records at the same time, but it was very rarely used in practice. It has been removed from the default CMS installation and has had its commercial support removed. Its integration support has been removed from `silverstripe/cms`, `silverstripe/admin`, and `silverstripe/asset-admin`, all of which provided "Add to campaign" UX functionality. This means that it may not be possible to get any sort of campaign admin functionality in CMS 6 even if using a forked version of `silverstripe/campaign-admin`.
+`silverstripe/campaign-admin` was initially going to loose commercial support, though this decision was later changed and `silverstripe/campaign-admin` will continue being commercially supported.
```

Comment:
```suggestion
```
I don't think that needs to be said, but if you do want to keep it, it should probably be said in the campaign admin section below.

Reply:
Removed